packages/api-client/package.json

```diff
@@ -19,7 +20,10 @@
         "lint": "yarn eslint",
         "lint:fix": "yarn eslint --fix && yarn prettier --write",
         "eslint": "eslint ./src --ext .js,.jsx,.ts,.tsx",
-        "prettier": "prettier --check '{src,public}/**/*.{js,jsx,ts,tsx,css,html,svg}'"
+        "prettier": "prettier --check '{src,public}/**/*.{js,jsx,ts,tsx,css,html,svg}'",
+        "npm-publish": "npm publish --access public",
+        "npm-version-patch": "npm version patch",
+        "npm-version-minor": "npm version minor"
```

Comment:
I want to revisit these commands longterm but we can add an issue for it.

Reply:
These might not be needed and we could switch to lerna publish if we switch over to npm workspaces instead of yarn workspaces because yarn has a known bug with publishing the dist folder.  I agree that we can look at this in the future.